tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "target": "ES6",
+        "module": "esnext",
+        "lib": ["es2017", "es7", "es6", "dom", "DOM.Iterable"],
+        "outDir": "build",
```

Comment:
This is the new bit, that was possibly missing previously to spit `App.tsx` into the build folder.